src/wtegTracking.js

```diff
@@ -0,0 +1,33 @@
+import{extractMetaTagContent} from './modules/extractMetaTagContent';
+import{promoObjFunc} from './modules/promoObj';
+import{pushInDataLayer} from './modules/pushInDataLayer';
+import{watermarkCheck} from './modules/discoveryServerSide/watermarkCheck';
+import{baseWTEGObjFunc} from './modules/baseWTEGPromoObj';
+
+// Wait for the DOM to load
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Hi @JamesWChan - why not use `document.getElementById` rather than `querySelector`?